recipes/libmemcached/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+set -x
+
+source activate "${CONDA_DEFAULT_ENV}"
+
+./configure --prefix="$PREFIX"
+
+make
+## tests hang on `testapp` on osx and linux
+# make check
```

Comment:
Ideally we should create a patch to skip that test and run the rest. But that is not a blocker to merge this.